systest/cloud/minio.env

```diff
@@ -0,0 +1,2 @@
+MINIO_ACCESS_KEY=accesskey
```

Comment:
there is a file that is already there in the dgraph folder. Can we just use that?